### PR TITLE
Triming IMAP and SMTP hosts on setup.

### PR DESCRIPTION
**Pull request by Ad5001**
Often when you try to add new mail accounts from manual setup, you copy the IMAP and SMTP hosts from your provider. However, more than often, copying and pasting like that may lead to additional unwanted whitespaces on the hosts fields.

This pull requests adds trimming to both IMAP and SMTP hosts on setup in the Accounts controller.

It doesn't cost much but is a great quality of life feature because the mail app tries, unsuccessfully, to connect to hosts with whitespaces and doesn't give any meaningful message related to it.

### CODE REVIEW

**Comment by ChristophWurst** [comment]
I'm fine with trimming. It's a nice enhancement. But I'd rather put this responsibility to the frontend because that is where we deal with user input. The controller method on the backend should expect clean input data.

Are you familiar with javascript development to move the logic?

**Comment by Ad5001** [comment]
While I'm not familiar with Vue.js, I'm familiar enough with javascript to move it there. I'll look into the format of the manualConfig variable when I can.

File: src/components/AccountForm.vue [hunk 1]
```diff
@@ -332,6 +332,11 @@ export default {
...
 				})
 			} else {
+				// Removing additional whitespaces from manual configuration hosts
+				// In order to avoid issues when copy pasting imap & smtp hosts from providers documentations,
+				// which may have whitespaces.
+				this.manualConfig.imapHost = this.manualConfig.imapHost.trim()
+				this.manualConfig.smtpHost = this.manualConfig.smtpHost.trim()
 				return this.save({
 					autoDetect: false,
 					...this.manualConfig,
```
Comment (ChristophWurst) [inline src/components/AccountForm.vue:342]:
```suggestion
				return this.save({
					autoDetect: false,
					...this.manualConfig,
					imapHost: this.manualConfig.imapHost.trim(),
					smtpHost: this.manualConfig.smtpHost.trim(),
```

same result but without the side effects

Comment (Ad5001) [reply src/components/AccountForm.vue:342]: I didn't know that syntax was possible. Added to the changes.

**Review by ChristophWurst — APPROVED** [review]

**Comment by ChristophWurst** [comment]
@Ad5001 DCO is not happy. Could you squash your commits into one and sign off that final commit? Thanks :v:

**Pull request merged**